test/models/host/content_facet_test.rb

```diff
@@ -197,6 +197,16 @@ def test_save_bound_repos_by_paths
       assert_equal content_facet.bound_repositories, [repo]
     end
 
+    def test_save_bound_repos_by_paths_same_path
+      content_facet.content_view = repo.content_view
+      content_facet.lifecycle_environment = repo.environment
+      content_facet.bound_repositories = [repo]
+
+      content_facet.update_repositories_by_paths(["/pulp/repos/#{repo.relative_path}"])
+
+      assert_equal content_facet.bound_repositories, [repo]
+    end
```

Comment:
@jlsherrill oh! i was not aware that was possible, that is much more readable. I will update